odd-platform-ui/src/components/Overview/DataEntityList/DataEntityList.tsx

```diff
@@ -23,39 +23,53 @@ const DataEntityList: React.FC<OverviewDataEntityProps> = ({
   const { dataEntityDetailsPath } = useAppPaths();
 
   return (
-    <Grid item>
+    <S.DataEntityListContainer item>
       <S.SectionCaption variant="h4" sx={{ mb: 2 }}>
         {entityListIcon}
         {entityListName}
       </S.SectionCaption>
+
       <S.ListLinksContainer>
         {dataEntitiesList.map(item => (
           <li key={item.id}>
-            <Grid container alignItems="center" wrap="nowrap">
-              <S.ListLink to={dataEntityDetailsPath(item.id)}>
+            <S.ListLink
+              to={dataEntityDetailsPath(item.id)}
+              className={item.hasAlerts ? 'has-alerts' : undefined}
```

Comment:
In our project now we try to avoid using classNames. We are using styled-components, so, pass `hasAlerts` like a prop to styled component. And depending on the props value you will apply backgroundColor or not.